apps/ui-layout/registry/components/linear-modal/index.tsx

```
@@ -52,16 +52,16 @@ export default function LinearCard() {
+                className='flex w-full flex-col overflow-hidden border bg-muted hover:bg-muted/80'
               >
                 <DialogImage src={item.url?.src} alt='' className='h-52 w-full object-cover' />
                 <div className='flex grow flex-row items-end justify-between p-3'>
                   <div>
-                    <DialogTitle className='text-zinc-950 text-xl dark:text-zinc-50'>
+                    <DialogTitle className='text-foreground text-xl'>
                       {item.title}
                     </DialogTitle>
                   </div>
-                  <button className='absolute bottom-2 right-2 p-2 dark:bg-neutral-800 bg-neutral-200 hover:bg-neutral-300 rounded-lg dark:hover:bg-neutral-900'>
+                  <button className='absolute bottom-2 right-2 p-2 bg-muted hover:bg-muted/80 rounded-lg'>
```

Biome (2.4.16)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)